jpype/_jcustomizer.py

```diff
@@ -121,10 +124,11 @@ def _applyStickyMethods(cls, sticky):
     for method in sticky:
         attr = getattr(method, '__joverride__')
         rename = attr.get('rename', None)
+        name = method.__name__
         if rename:
-            type.__setattr__(
-                cls, rename, type.__getattribute__(cls, method.__name__))
-        type.__setattr__(cls, method.__name__, method)
+            orig = type.__getattribute__(cls, name)
+            type.__setattr__(cls, rename, orig)
+        type.__setattr__(cls, name, method)
```

Comment:
Is it intended to have the original name and the rename pointing to the same method? It would be an alias then, right?

Reply:
Not sure what you mean here...

```
 orig = type.__getattribute__(cls, name)    # orig <= cls[name]
 type.__setattr__(cls, rename, orig)            # cls[rename] <= orig
 type.__setattr__(cls, name, method)         # cls[name]<=method
```

So the original is renamed and then the method from the prototype put in place.